relay/README.md

```diff
@@ -210,5 +237,11 @@ It lives in the go-libp2p package and is named `/ipfs/relay/line/0.1.0`.
 - Capable of *accepting* connections, and *relaying* connections.
 - Not capable of *connecting* via relaying.
 
-Since the existing protocol is incomplete, insecure, and certainly not used,
-we can safely remove it.
+Since the existing protocol is incomplete, insecure, and certainly not used, we can safely remove it.
+
+## Future work
+
+We have considered more features but won't be adding them on the first iteration of Circuit Relay, the features are:
+
+- Multihop relay - With this specification, we are only enabling single hop relays to exist. Multihop relay will come at a later stage as Packet Switching.
```

Comment:
The js implementation does implement multihop dialing in the form of `shallot/matreshka` dialing. This was based on the conversations we had at the time of me starting the implementation, what has changed that we now have scrapped this part?